package.json

```diff
@@ -77,6 +83,11 @@
     "htmltidy": "0.0.6",
     "loose-envify": "^1.2.0",
     "mocha": "^2.3.4",
+    "react": "^15.4.0",
+    "react-dom": "^15.4.0",
+    "uglify-js": "^3.0.15",
+    "uglifyify": "^3.0.4",
+    "webpack": "^2.6.1",
```

Comment:
I don't believe webpack is currently listed as an explicit dependency in package.json.  I don't believe it's necessary to include here, but let me know if you're having a problem without it.